onebusaway-android/src/main/java/org/onebusaway/android/nav/FeedbackReceiver.java

```diff
@@ -0,0 +1,135 @@
+package org.onebusaway.android.nav;
+
+import android.app.NotificationManager;
+import android.content.BroadcastReceiver;
+import android.content.Context;
+import android.content.Intent;
+import android.os.Bundle;
+import android.util.Log;
+
+import org.apache.commons.io.FileUtils;
+import org.onebusaway.android.R;
+import org.onebusaway.android.app.Application;
+
+import java.io.File;
+import java.io.IOException;
+
+import androidx.core.app.NotificationCompat;
+import androidx.core.app.RemoteInput;
+
+import static org.onebusaway.android.nav.NavigationService.LOG_DIRECTORY;
+
+
+public class FeedbackReceiver extends BroadcastReceiver {
+    public static final String TAG = "FeedbackReceiver";
+
+    public static final String TRIP_ID = ".TRIP_ID";
+    public static final String NOTIFICATION_ID = ".NOTIFICATION_ID";
+    public static final String CALLING_ACTION = ".CALLING_ACTION";
+    public static final String ACTION_NUM = ".ACTION_NUM";
+
+    public static final int FEEDBACK_NO = 1;
+    public static final int FEEDBACK_YES = 2;
+
+    public static final int DISMISS_FEEDBACK_NOTIFICATION = 1;
```

Comment:
This should be changed to an Intent action:

`public static final String ACTION_DISMISS_FEEDBACK =
            BuildConfig.APPLICATION_ID + ".action.DISMISS_FEEDBACK";`

Reply:
Modified `public static final int DISMISS_FEEDBACK_NOTIFICATION` to `ACTION_DISMISS_FEEDBAC`